tests/Perf/Microsoft.Identity.Web.Perf.Client/TestRunner.cs

```diff
@@ -129,6 +123,13 @@ private async Task<AuthenticationResult> AcquireTokenAsync(int userIndex)
             var scopes = new string[] { _configuration["ApiScopes"] };
             var upn = $"{NamePrefix}{userIndex}@{_configuration["TenantDomain"]}";
 
+            var _msalPublicClient = PublicClientApplicationBuilder
```

Comment:
Recreating a PublicClientApplication each time to avoid the MSAL.NET performance bug which makes the cache grow with the number of users even if we store it in a different file (singleton)